### PR TITLE
asadiqbal08/added resource pages in CMS

### PR DESCRIPTION
**Pull request by asadiqbal08**
#### What are the relevant tickets?
[ticket](https://trello.com/c/hIdWN0H3/76-terms-of-service-page)

#### What's this PR do?
Added generic resource page structure.

#### How should this be manually tested?
Try to create some resource pages e.g. Privacy Policy, Term of Service etc. 

#### Screenshot: 
Screen Shot 2020-05-04 at 2 57 35 PM

##### NOTE: As we do not have much info about design so keep it in mind for future changes in corresponding scss, images and templates structure.

### CODE REVIEW

**Comment by annagav** [comment]
@asadiqbal08, you have some conflicts that need to be resolved.

File: cms/models.py [hunk 1]
```diff
@@ -38,3 +39,33 @@ def get_context(self, request, *args, **kwargs):
         context = super(BootcampPage, self).get_context(request)
         context["title"] = self.title
         return context
+
+
+class ResourcePage(Page):
+    """
+    Basic resource page for all resource page.
+    """
+    template = "../../bootcamp/templates/resource_template.html"
```
Comment (annagav) [inline cms/models.py:48]: This path seems to be incorrect

Comment (asadiqbal08) [reply cms/models.py:48]: loading fine at my end. further elaborate ?

Comment (asadiqbal08) [reply cms/models.py:48]: updated: I think, something has been updated in settings.py file after rebase. anyway, I will do the corresponding fix.

Comment (annagav) [reply cms/models.py:48]: Screen Shot 2020-04-30 at 10 11 24 PM

Comment (umarmughal824) [reply cms/models.py:48]: Yes @annagav you are right. @asadiqbal08 gavin has renamed the base app from `bootcamp` to `main` so all your bootcamp app changes should be in `main` app. That's the issue here.

File: static/scss/resource.scss [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    text-decoration: none;
+
+    &:after {
+      content: "\e905";
+      font-family: icomoon;
+      margin-left: 10px;
+    }
+  }
+}
+
```
Comment (umarmughal824) [inline static/scss/resource.scss:86]: It should be an empty line but now its two characters at the end of the file.

**Pull request merged**